cmds/exp/tcpdump/gen.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:build ignore
+
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func main() {
+	file, err := os.Create("well_known_ports.go")
+	if err != nil {
+		panic(err)
+	}
+	defer file.Close()
+
+	fmt.Fprintln(file, `// Copyright 2024 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+var wellKnownPortsMap = map[string]string{`)
```

Comment:
this would be a good place to read /etc/services, as long as you go to this trouble. 
/etc/services
has been a well known file for almost a half  century, so it's not like it's going anywhere. Only Plan 9 no longer has it :-)

Reply:
Sure, just hadn't come that far. Check the latest commit for the version generated with /etc/services.